src/utils/logoutput.cpp

```diff
@@ -3,16 +3,15 @@
 
 #include <stdio.h>
 #include <stdarg.h>
+#include <mutex>
 
-#ifdef _OPENMP
-    #include <omp.h>
-#endif // _OPENMP
 #include "logoutput.h"
 
 namespace cura {
 
 static int verbose_level;
 static bool progressLogging;
+static std::mutex log_mutex;

```

Comment:
Added a global mutex for standard output synchronization. Mainly to make `-fsanitize=thread` happy and less verbose.
This is not critical and can be omitted.